src/commands/autocomplete/index.ts

```diff
@@ -35,15 +39,33 @@ export default class Index extends AutocompleteBase {
     if (!flags['refresh-cache']) {
       const bin = this.config.bin
       const tabStr = shell === 'bash' ? '<TAB><TAB>' : '<TAB>'
-      const note = shell === 'zsh' ? `After sourcing, you can run \`${chalk.cyan('$ compaudit -D')}\` to ensure no permissions conflicts are present` : 'If your terminal starts as a login shell you may need to print the init script into ~/.bash_profile or ~/.profile.'
+
+      const instructions = shell === 'powershell' ?
+        `New-Item -Type Directory -Path (Split-Path -Parent $PROFILE) -ErrorAction SilentlyContinue
```

Comment:
This will create the dir to the $PROFILE file and silently continue if it already exists:
```powershell
PS /> echo $PROFILE
/root/.config/powershell/Microsoft.PowerShell_profile.ps1
PS /> New-Item -Type Directory -Path (Split-Path -Parent $PROFILE) -ErrorAction SilentlyContinue

    Directory: /root/.config

...
```

`/root/.config/powershell` is created.